packages/block-editor/src/components/use-on-block-drop/index.js

```diff
@@ -292,7 +295,8 @@ export default function useOnBlockDrop(
 					{
 						layout: {
 							type: 'flex',
-							flexWrap: areAllImages ? null : 'nowrap',
+							flexWrap:
+								areAllImages && galleryBlock ? null : 'nowrap',
```

Comment:
I believe we only want to set `null` for the gallery block.

Reply:
Yes, I believe that's correct.